modules/virt-autoupdate-custom-bootsource.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-autoupdate-custom-bootsource_{context}"]
+= Enabling automatic updates on custom boot sources
+
+{VirtProductName} automatically updates pre-defined boot sources by default. You must manually enable automatic imports and updates on any custom boot sources.
+
+.Procedure
+
+. Use the following command to enable automatic updates on a custom boot source:
++
+[source,terminal]
+----
+$ oc apply -n openshift-cnv -f add-custom-boot-sources.yaml
+----
+
++
+.Example in CentOS 7
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  dataImportCronTemplates:
+  - metadata:
+      name: centos7-image-cron
+      annotations:
+        cdi.kubevirt.io/storage.bind.immediate.requested: "true" <1>
+    spec:
+      schedule: "0 */12 * * *" <2>
+      template:
+        spec:
+          source:
+            registry: <3>
+              url: docker://quay.io/containerdisks/centos:7-2009
```

Comment:
I think `"pullMethod": "node"` is also needed; @arnongilboa ?

Reply:
No, better use the default (`pod` pullMethod) in this case. `node` pullMethod is based on the node docker cache, which is useful when registry image is usable via Container.Image but CDI importer is not authorized to access it (e.g. registry.redhat.io requires a pull secret).

See [here](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/image-from-registry.md#import-registry-image-into-a-data-volume-using-node-docker-cache)

Reply:
@arnongilboa Since they should use the default, is there anything to add/edit here, or is this ok as-is?